frontend/src/scenes/insights/filters/BreakdownFilter/TaxonomicBreakdownFilter.tsx

```diff
@@ -61,7 +62,13 @@ export function BreakdownFilter({
                           } else {
                               const newParts = breakdownParts.filter((_, i) => i !== index)
                               setFilters({
-                                  breakdowns: newParts.map((np): Breakdown => ({ property: np, type: breakdown_type })),
+                                  breakdowns: newParts.map(
+                                      (np): Breakdown => ({
+                                          property: np,
+                                          type: breakdown_type,
+                                          normalize_url: isURLNormalizeable(np.toString()),
```

Comment:
Will we send this property over with other non-url breakdown params as well? I'd prefer then to not send the key at all.

Reply:
Yep, https://github.com/PostHog/posthog/pull/12808/files#diff-5acf0d9fad9469e30132e1a2ef62c761a5f2837c1a0da79670b14fb4d6c8af2cR122